pkg/binary/envoy/runtime.go

```diff
@@ -139,8 +150,9 @@ func (r *Runtime) GetPid() (int, error) {
 // envoyReady reads the HTTP status from the /ready endpoint and returns binary.StatusReady on 200 or
 // binary.StatusInitializing on 503.
 func (r *Runtime) envoyReady() (int, error) {
```

Comment:
note: I noticed a lot of defense in this codeblock, but it is only used in tests, so has no risk of filling logs. To improve coverage and focus I'll remove this whole status thing in another PR